tests/optimization/test_optimization.py

```diff
@@ -17,6 +17,9 @@
 import os
 import tempfile
 import unittest
+from typing import Type
+
+from torch.optim.lr_scheduler import LambdaLR
```

Comment:
This import should be removed to avoid a error when PyTorch is not installed. It's not really necessary anyway.